pkg/appAPIs/appAPIs.go

```diff
@@ -129,15 +131,50 @@ func (cli_api *AppAPI) createAppAPI(createInfo string, token string) ([]byte, er
 
 // To get all the apps information.
 func CreateApp(name string, image string, username string, password string,
-	env []string, port string, token string) error {
+	env []string, envFilePath string, port string, token string) error {
 	// Endpoint to list apps.
 	url := fmt.Sprintf(constants.APPURL)
 	var createInfo string
-	if env != nil {
+	// if both are provided then we will take union of two
+	if env != nil && envFilePath != "" {
+		sliceFromEnv, sliceMap := genEnvSlice(env)
+		sliceFromEnvFile, envMap, err := GetSliceFromEnvFile(envFilePath)
+
+		if err != nil {
+			return fmt.Errorf("%s", err)
+		}
+
+		// if a same key is found in both (envFile and env from command line, we will throw an error)
```

Comment:
Let's add a detailed comments before all  cases. 
NiT: If the same key is found in both the `.env` file passed through --envPath and the --env flag.